### ISSUE TITLE
About the training details.

### ISSUE THREAD

**Issue by bb12346** [wty-ustc/HairCLIP#6]
Thank you for your great project!

In this paper, you said “We train and evaluate our hair mapper on the CelebA-HQ dataset. Since we use e4e [43] as our inversion encoder, we follow its division of the training set and test set.” However, I found that e4e used the FFHQ dataset for training and the CelebA-HQ test dataset for evaluation. Hence, I feel confused.
My question is that how to split the training and test datasets on the CelebA-HQ dataset?

**Comment by wty-ustc** [comment]
e4e is an inversion method that predicts latent codes that are more suitable for editing. e4e is trained using FFHQ and tested using CelebA-HQ  which is exactly correct. We followed the [StyleCLIP](https://github.com/orpatashnik/StyleCLIP) method to invert the latent codes of CelebA-HQ with e4e and split them into training and testing parts, in the same way as StyleCLIP. You can download latent codes from [train set](https://drive.google.com/file/d/1gof8kYc_gDLUT4wQlmUdAtPnQIlCO26q/view?usp=sharing), [test set](https://drive.google.com/file/d/1j7RIfmrCoisxx3t-r-KC02Qc8barBecr/view?usp=sharing).

**Comment by bb12346** [comment]
Thank you for your reply.
